source/includes/connection-write-config.rst

```diff
@@ -0,0 +1,34 @@
+Specifying Properties in ``connection.uri``
+-------------------------------------------
+
+If you use :ref:`SparkConf <spark-conf>` to specify any of the previous settings, you
+can either list them individually or include them in the ``connection.uri`` setting.
+
+The following code example shows how to specify the ``connection.uri``,
+``database``, ``collection``, and ``convertJson`` settings individually in a
+``SparkConf`` configuration file:
+
+.. code:: cfg
+
+   spark.mongodb.write.connection.uri=mongodb://127.0.0.1/
+   spark.mongodb.write.database=test
+   spark.mongodb.write.collection=myCollection
+   spark.mongodb.write.convertJson=any
+
+Instead, you can specify these settings in the value of ``connection.uri``:
```

Comment:
(copying my earlier comment here so it's seen on both pages)

Q: Is this the preferred/recommended way to set the settings as opposed to doing so individually? I think leading in with "Instead, you can..." makes it sound like the previous option is not as good as this one.